src/data/STATUSES/index.js

```diff
@@ -67,8 +67,76 @@ function correctIdsToMatchLogs (obj) {
 		} else {
 			status.id = status.id + STATUS_ID_OFFSET
 		}
+		if (status.stackWith) {
+			status.stackWith += STATUS_ID_OFFSET
+		}
 	})
 	return obj
 }
 
+export const STATUS_EFFECT_TYPES = {
```

Comment:
blocking;

I'm well on-record as not being particularly on-board with classifying statuses - but if people are _determined_ for it to be a thing, then I think it's going to be necessary to strike up a conversation about _how_ we do it - and I don't think this current structure is going to be it.

Given it (looks?) pretty unused currently, can we nuke the structure + usage of it in the short term, and re-visit adding it in a future PR? Would help reduce the surface of this PR, too - as it's pretty damn big.